src/wp-includes/class-wp-block-bindings-source.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Block Bindings API: WP_Block_Bindings_Source class.
+ *
+ *
+ * @package WordPress
+ * @subpackage Block Bindings
+ * @since 6.5.0
+ */
+
+/**
+ * Class representing block bindings source.
+ *
+ * This class is designed for internal use by the Block Bindings registry.
+ *
+ * @since 6.5.0
+ * @access private
+ *
+ * @see WP_Block_Bindings_Registry
+ */
+final class WP_Block_Bindings_Source {
+
+	public function __construct( string $name, array $source_properties ) {
```

Comment:
PHPDoc for the constructor is missing.

By convention, all class fields go first, so the constructor should also get moved before `get_value` function.

Reply:
Done in 9853934296fa166ceeaa1c0f6d03797f9f3a712d & 16641c0fcda30bd7306192fd89e48871c93d6dc8